PhoneNumberKit/PhoneNumberKit.swift

```diff
@@ -316,7 +316,7 @@ public final class PhoneNumberKit: NSObject {
     ///
     /// - returns: an optional Data representation of the metadata.
     public static func defaultMetadataCallback() throws -> Data? {
-        let frameworkBundle = Bundle(for: PhoneNumberKit.self)
+        let frameworkBundle = Bundle.module
```

Comment:
May I suggest this, I think this will provide backwards compatibility.
```suggestion
        #if SWIFT_PACKAGE && swift(>=5.3)
        let frameworkBundle = Bundle.module
        #else
        let frameworkBundle = Bundle(for: PhoneNumberKit.self)
        #endif
```